internal/project/deploy.go

```diff
@@ -48,6 +49,10 @@ var DeployCommand = &command.Command{
 	RunS:  deploy,
 }
 
+func updateWithPrompt(existing []byte, new []byte) bool {
```

Comment:
no need to redeclare this. You can just use the `output.ShowContractDiffPrompt` directly.

Reply:
Done